Dockerfile

```
@@ -0,0 +1,16 @@
+# A simple container for variant-service.
+# Runs service on port 80.
+# Healthchecks service up every 5m.  
+
+FROM python:3.10
```

Probably good to use the latest Python version
```suggestion
FROM python:3.12
```

---

we can upgrade the python version to 3.12 , however this would depend on the compatibility with all the dependant modules of Variation Normalizer. If currently Variation Normalizer is working on 3.12 , we can go move to 3.12.

---

Yes, Variation Normalizer works with 3.12